src/devices/zemismart.ts

```diff
@@ -355,11 +355,18 @@ const definitions: Definition[] = [
         model: 'ZMO-606-S2',
         vendor: 'Zemismart',
         description: 'Smart 2 gangs switch with outlet',
-        extend: [
-            deviceEndpoints({endpoints: {l1: 1, l2: 2, l3: 3}}),
-            identify(),
-            tuya.modernExtend.tuyaOnOff({indicatorMode: true, onOffCountdown: true, endpoints: ['l1', 'l2', 'l3']}),
-        ],
+        extend: [tuya.modernExtend.tuyaOnOff({powerOutageMemory: true, indicatorMode: true, onOffCountdown: true, endpoints: ['l1', 'l2', 'l3']})],
+        endpoint: (device) => {
+            return {l1: 1, l2: 2, l3: 3};
+        },
+        configure: async (device, coordinatorEndpoint) => {
```

Comment:
I think the only change needed to fix this issue is: `configure: tuya.configureMagicPacket,`

Reply:
Thanks, I changed to use configure: tuya.configureMagicPacket but I had to keep the enpoint: (device) in line 359. Without this I would get the error "No converter available for 'state_l1' ("OFF")"

Reply:
That's interesting because that's what `deviceEndpoints({endpoints: {l1: 1, l2: 2, l3: 3}}),` should do, did you check with z2m 1.39.0?

Reply:
Oh I see what happened, the "No converter available for 'state_l1' ("OFF")" happened because I removed the `deviceEndpoints({endpoints: {l1: 1, l2: 2, l3: 3}}),` to include the `endpoint: (device) => {`. When I removed everything and changed to `configure: tuya.configureMagicPacket,`, the code was missing and throwing this error. Now everything works as expected... just pushed an update.